tfhub_dev/assets/intel/midas/v2_1/1/1.md

```diff
@@ -0,0 +1,83 @@
+# Module intel/midas/v2_1/1
```

Comment:
In the current state of the PR, `intel/midas/v2_1/1` and `intel/midas/v2_1/2` represent two versions of the same model. There might be a more appropriate TF Hub concept we can use to represent the relationship between these models. What is the difference between `intel/midas/v2_1/1` and `intel/midas/v2_1/2`.